docs/guide/README.md

```diff
@@ -191,7 +191,7 @@ algob -h deploy
 You can write your scripts and tests in TS. To initialize a new typescript project add `--typescript` to the init flag. Example, if you follow the [Create an algob project](#create-an-algob-project) section then:
 
 ```shell
-yarn run algob init --typescript
+yarn run algob init <path-where-to-create> --typescript
```

Comment:
Should we add some example to make more clearly?

Reply:
agree

Reply:
Done. Thanks